Localization/de-DE/home.yaml

```
@@ -2,30 +2,42 @@
 url: "https://foldingathome.org/"
 language: "de-DE"
 text: [
+# I AM One IN A MILLION bold text
+#"I AM One IN A MILLION",
+"ICH BIN Einer UNTER MILLIONEN",
+
+# Associated text
+#"Regardless if you are already folding or haven't heard a word about it before, we need your help to reach our goal -- which is 1 million folders.",
+"Egal, ob Sie bereits falten oder noch kein Wort davon gehört haben: Wir brauchen Ihre Hilfe, um unser Ziel - eine Million Faltende - zu erreichen",
```

```suggestion
"Egal, ob Sie bereits falten oder noch kein Wort davon gehört haben: Wir brauchen Ihre Hilfe, um unser Ziel – eine Million Faltende – zu erreichen",
```
gedankenstrich instead of bindestrich

---

```suggestion
"Egal, ob Sie bereits mitmachen oder noch nie davon gehört haben: Wir brauchen IHRE Hilfe, um unser Ziel - eine Million Unterstützer - zu erreichen",
```
- _falten_ und _Faltende_ sound very embarrassing to me. Furthermore we used _Unterstützer_ on other pages already
- Uppercase IHRE to address the reader more
- _noch nie_ is more common imho

---

See Discord, "Faltende" sounded weird, but was an attempt to stick closer to the English.
I do agree with your suggestions 100% though, "falten"/"Faltende"/"Falter" sounds a bit like Google Translate ;)